docs/masthead.md

```diff
@@ -65,3 +96,21 @@ If you’re using Nunjucks macros in production with `html` options, or ones end
 | :------ | :----- | :---------------------------------------------------------------------- |
 | **src** | string | URL of image displayed on the right of the masthead on desktop layouts. |
 | **alt** | array  | Alternative text for image.                                             |
+
+## Research on this component
+
+This component is based on the Masthead component used in the [Product Page Example](https://github.com/alphagov/product-page-example). This in turn was based on the product pages for:
+
+- [GOV.UK Design System](https://design-system.service.gov.uk)
+- [GOV.UK Forms](https://www.forms.service.gov.uk)
+- [GOV.UK Notify](https://www.notifications.service.gov.uk)
+- [GOV.UK One Login](https://www.sign-in.service.gov.uk)
+- [GOV.UK Pay](https://www.payments.service.gov.uk)
+
+Versions of this component are currently being used on these services:
+
+- [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk)
```

Comment:
Some more examples (too many?)

```suggestion
- [Register trainee teachers](https://www.register-trainee-teachers.service.gov.uk)
- [Gender pay gap service](https://gender-pay-gap.service.gov.uk)
- [Find a Learning Aim](https://submit-learner-data.service.gov.uk/find-a-learning-aim/)
- [Ethnicity facts and figures](https://www.ethnicity-facts-figures.service.gov.uk)
- [HMRC Developer Hub](https://developer.service.hmrc.gov.uk/api-documentation)
- [Find planning and housing data in England](https://www.planning.data.gov.uk/)
```

Reply:
It really ought to be a component in the Design System by now. It (or variations of it) are everywhere!